bitcoin/src/blockdata/fee_rate.rs

```diff
@@ -136,3 +136,68 @@ impl Div<Weight> for Amount {
 }
 
 crate::parse::impl_parse_str_from_int_infallible!(FeeRate, u64, from_sat_per_kwu);
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::u64;
+
+    #[test]
+    fn fee_rate_const_test() {
+        assert_eq!(0, FeeRate::ZERO.to_sat_per_kwu());
+        assert_eq!(u64::min_value(), FeeRate::MIN.to_sat_per_kwu());
+        assert_eq!(u64::max_value(), FeeRate::MAX.to_sat_per_kwu());
+        assert_eq!(250, FeeRate::BROADCAST_MIN.to_sat_per_kwu());
+        assert_eq!(750, FeeRate::DUST.to_sat_per_kwu());
```

Comment:
I'm not really sure if these are useful.

Reply:
Since there is not any interaction between the associated methods in this mod, it's not really possible to otherwise test the behavior.

Reply:
I mean we probably don't need to test constants at all.

Reply:
maybe it's pedantic but it can help to find bugs imo.  Also tests can help other developers understand the code behavior and know how to use the lib.